packages/utils/src/signatureUtils.ts

```diff
@@ -1,10 +1,8 @@
-import assert = require('assert')
```

Comment:
This was causing import errors in some places and it's not really necessary here, cutting.